src/handle/command.ml

```diff
@@ -435,21 +435,16 @@ let get_proof_data : compiler -> sig_state -> p_command -> cmd_output =
             if finished ps then
               ( wrn pe.pos "The proof is finished. Use 'end' instead."; ss )
             else
-              match ps.proof_term with
-              | Some m when opaq ->
-                  (* We admit the initial goal only. *)
-                  Tactic.admit_meta ss p_sym_nam.pos m
-              | _ ->
-                  (* We admit all the remaining typing goals. *)
-                  let admit_goal ss g =
-                    match g with
-                    | Unif _ -> ss
-                    | Typ gt ->
-                        let m = gt.goal_meta in
-                        match !(m.meta_value) with
-                        | None -> Tactic.admit_meta ss p_sym_nam.pos m
-                        | Some _ -> ss
-                  in List.fold_left admit_goal ss ps.proof_goals
+              (* We admit all the remaining typing goals. *)
+              let admit_goal ss g =
+                match g with
+                | Unif _ -> fatal pos "Remaining unification goals."
```

Comment:
It's OK by me as it is, but aren't there cases where unification goals may be solved by user supplied unification rules in the proof?